frontend/src/pages/projects/screens/detail/notebooks/__tests__/useNotebookImageData.spec.ts

```diff
@@ -106,4 +106,21 @@ describe('getNotebookImageData', () => {
     const result = getNotebookImageData(notebook, images);
     expect(result?.imageAvailability).toBe(NotebookImageAvailability.DELETED);
   });
+
+  it('should fail without fix', () => {
+    const imageName = 'jupyter-datascience-notebook';
+    const tagName = '2024.1';
+    const notebook = mockNotebookK8sResource({
+      lastImageSelection: `${imageName}:${tagName}`,
+      image: `image-registry.openshift-image-registry.svc:5000/opendatahub/${imageName}:${tagName}`,
```

Comment:
Your test case shouldn't include an image path from the `image-registry`. This bug happens when the internal image registry is disabled.
This test case passes even without your change therefore it's not testing the correct scenario.